### DISCUSSION TITLE
Is the GUI supported yet on MacOS on Apple Silicon?

### DISCUSSION THREAD

**Discussion by esba1ley** [snl-dakota/dakota#126] [Q&A] [answered]
I tried to run dakota_gui.sh after installing version 6.20 on Apple silicon, and got error messages saying things were broken.  Is the GUI supported on Apple Silicon yet?

**Comment by jadamstephens** [comment, marked as answer]
This is likely a code signing issue.

In a terminal, change to the folder that contains the Dakota install. Run the command: `xattr -cr <DAKOTA FOLDER>`, substituting the name of the folder for `<DAKOTA FOLDER>`.

**Comment by esba1ley** [comment]
That did it.  Did I use the wrong installation instructions?  the <DAKOTA FOLDER>/share/dakota/INSTALL doesn't contain the string "xattr" in it at all.
